test/unit/spec/PrimaryAuth_spec.js

```diff
@@ -91,9 +91,10 @@ function (Q, OktaAuth, LoginUtil, Okta, Util, AuthContainer, PrimaryAuthForm, Be
     var beacon = new Beacon($sandbox);
     router.on('afterError', afterErrorHandler);
     if (refreshState) {
+      var stateToken = 'aStateToken';
       Util.mockRouterNavigate(router);
       setNextResponse(resUnauthenticated);
-      router.refreshAuthState('aStateToken');
+      router.refreshAuthState(stateToken);
```

Comment:
nit: this also could just remain as it was before, no other place is reusing the created `stateToken` variable